docs/basic-features/script.md

```diff
@@ -174,6 +174,26 @@ export default function Home() {
 }
 ```
 
```

Comment:
@molebox This should probably be its own section, right?

Reply:
I put it with the Script Loading stuff. Happy to move wherever.